src/CSSBattle/3_PushButton/ZhangZhikai.html

```diff
@@ -0,0 +1,46 @@
+<div class="rect" >
+  <div class="circle outer">
+    <div class="circle inner" />
+  </div>
+  </ div>
```

Comment:
Consider removing unnecessary indentation and spacing
```suggestion
</div>
```

Reply:
50ed92b92dcb6d7fac9a9b6965067d8e1abde559